docs/pages/server-access/rbac.mdx

```diff
@@ -178,6 +178,9 @@ spec:
       # Optional: Session recording mode for SSH sessions.
       # If not set, the value set on default will be used.
       ssh: best_effort|strict
+    # require_session_mfa requires per-session MFA for any owner of this role.
```

Comment:
Is "owner" the right term here? Are you referring to users to whom this role is applied?

Reply:
That's the same language copied from https://goteleport.com/docs/access-controls/reference/.  Basically who's assigned the role.  We can change in both places assigned is better.